File: tests/fixtures/v2-sql-flow/positive/sql_flow.go
package sqlflow

func direct(r *Request, db *DB) {
	db.Query(r.URL.Query().Get("direct"))
}

func propagated(r *Request, db *DB) {
	query := r.URL.Query().Get("propagated")
	alias := query
	db.Query(alias)
}

File: tests/fixtures/v2-sql-flow/positive/SqlFlow.java
class SqlFlow {
    Object direct(HttpServletRequest request, Statement statement) throws Exception {
        return statement.executeQuery(request.getParameter("direct"));
    }

    void propagated(HttpServletRequest request, Statement statement) throws Exception {
        String query = request.getParameter("propagated");
        String alias = query;
        statement.executeQuery(alias);
    }
}
